js/ui/map.js

```diff
@@ -287,6 +287,50 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set the map's minimum zoom level, and zooms map to that level if it is currently below it. If no parameter provided, unsets the current minimum zoom (sets it to 0)
+     * @param {zoom} any number between 0 and 20
+     * @returns {Map} `this`
+     */
+    setMinZoom: function(minZoom) {
+
+        var zoom = minZoom === undefined ? 0 : minZoom;
```

Comment:
Agreed -- it's a compromise between clear and simple. More importantly, though, its consistent with the rest of our API.